Lib/test/test_ast.py

```diff
@@ -167,6 +198,10 @@ def to_tuple(t):
     "@a.b.c\ndef f(): pass",
     # Simple assignment expression
     "(a := 1)",
+    # Assignment expression in if statement
```

Comment:
Reason for these?

Reply:
In `if` and `while` statements, `Assignment Expression` can be used without parentheses, in other cases - no